server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js

```diff
@@ -37,7 +37,7 @@ async function uploadManuscript(_, { file, id, fileSize }, { user }) {
     manuscriptId: id,
     url: `manuscripts/${id}`,
     filename,
-    type: 'MANUSCRIPT_SOURCE_PENDING',
+    type: 'MANUSCRIPT_SOURCE',
```

Comment:
@jyenes Can you remove the changes to this file as its going to hide the existing problem